src/gwt/src/org/rstudio/core/client/command/ShortcutManager.java

```diff
@@ -471,6 +475,28 @@ public void execute()
       
       keyBuffer_.add(keyCombination);
       
+      // we handle `Ctrl+L` for the sublime mode here
+      // because it is conflicted with `consoleClear`
+      if (editorMode_ == KeyboardShortcut.MODE_SUBLIME)
+         {
+            if (keyCombination.getKeyCode() == KeyCodes.KEY_L &&
+                keyCombination.getModifier() == KeyboardShortcut.CTRL)
+         {
+            Element target = Element.as(event.getEventTarget());
```

Comment:
We need this because otherwise https://github.com/rstudio/rstudio/blob/5a7c762642ba4f267af29764f27697d9b779d35b/src/gwt/src/org/rstudio/core/client/command/ShortcutManager.java#L515-L532 will steal the `ctrl+L` key combination.